### PR TITLE
EOS-22276 default values removed for dns_servers and search_domains

### PR DESCRIPTION
**Pull request by zahidsk**
Signed-off-by: zahid.shaikh <zahid.shaikh@seagate.com>

### CODE REVIEW

File: lr-cli/cortx_setup/commands/node/prepare/network.py [hunk 1]
```diff
@@ -70,14 +70,12 @@ class NodePrepareNetwork(Command):
             'type': str,
             'nargs': '+',
             'optional': True,
-            'default': "",
             'help': 'DNS server'
```
Comment (83bhp) [inline lr-cli/cortx_setup/commands/node/prepare/network.py:73]: Please update help to:
'List of DNS servers for the provided network, given space-separated'

File: lr-cli/cortx_setup/commands/node/prepare/network.py [hunk 2]
```diff
@@ -70,14 +70,12 @@ class NodePrepareNetwork(Command):
...
         },
         'search_domains': {
             'type': host,
             'nargs': '+',
             'optional': True,
-            'default': "",
             'help': 'Search domain list'
```
Comment (83bhp) [inline lr-cli/cortx_setup/commands/node/prepare/network.py:79]: Please update help to:
'List of search domains for provided network, given space-separated'

**Review by 83bhp — CHANGES_REQUESTED** [review]

File: lr-cli/cortx_setup/commands/node/prepare/network.py (hunk 1 above)
Comment (zahidsk) [reply lr-cli/cortx_setup/commands/node/prepare/network.py]: Updated help message

**Review by 83bhp — APPROVED** [review]
Looks good, please  let know once the patch is tested and ready to be merged.

**Review by mazinamdar — APPROVED** [review]

**Pull request merged**